deploy/base/config.yaml

```
@@ -7,3 +7,4 @@ metadata:
     security-profiles-operator/config: ""
 data:
   SPOdImagePullPolicy: Always
+  EnableSelinux: "false"
```

Can we directly use a bool here?
```suggestion
  EnableSelinux: false
```

---

Haha, yes, I'll see about fixing this

---

Hmm, I must be misunderstanding something. I'm getting errors trying to use a straight-up boolean in a configmap, seems like CMs only allow string values?